docs/guides/execution-modes.mdx

```diff
@@ -5,73 +5,169 @@ description: An overview of the available execution modes in Qiskit Runtime; ses
+
+For full details about these values, including how to determine the ITTL value, review the [Maximum execution time guide.](/guides/max-execution-time)
+
+<Admonition>
+The time your workload actively runs is probably different from the amount of wall clock time a workload takes.
+</Admonition>
+
+## Basic workflow
+
+The basic workflow for batches and sessions is similar:
 
-[**Batch mode**](run-jobs-batch): A multi-job manager for efficiently running an experiment that is comprised of bundles of independent jobs. Use batch mode to submit multiple primitive jobs simultaneously.   To run in batch mode, specify `mode=batch` when instantiating a primitive or run the job in a batch context manager. See [Run jobs in a batch](run-jobs-batch) for examples.
+1. The first job in a batch or session enters the normal queue. Note: for batches, the entire batch of jobs is scheduled together.
+2. When the first job starts running, the TTL timer starts (and does not stop or pause until the end is reached).
+3. The ITTL timer starts after each job is completed.  If there are no workload jobs ready within the ITTL window, the workload is temporarily deactivated and normal job selection resumes. A job can reactivate the deactivated workload if the batch or session has not reached its TTL value.
+   <Admonition type="note">
+   The job must go through the normal queue to reactivate the workload.
+   </Admonition>
+3. If the TTL value is reached, the workload ends and any remaining queued jobs fail. However, any jobs already running will run to completion.
 
-[**Session mode**](sessions): A dedicated window for running a multi-job workload. This allows users to experiment with variational algorithms in a more predictable way and even run multiple experiments simultaneously, taking advantage of parallelism in the stack. Use sessions for iterative workloads or experiments that require dedicated access.  To run in session mode, specify `mode=session` when instantiating a primitive, or run the job in a session context manager. Learn more about sessions in the [Introduction to sessions,](/guides/sessions) and see [Run jobs in a session](run-jobs-session) for examples.
+The following video illustrates the session basic workflow, using sessions as an example:
+
+<video title="A user starts a session job and becomes the priority user.  They submit jobs to the QPU while other users wait. After the prioritized user's session finishes, the next person's jobs can begin processing." className="max-w-auto h-auto" controls>
+  <source src="/videos/guides/sessions/demo.mp4" type="video/mp4"></source>
+</video>
 
 <span id="sessions-versus-batch-usage" />
 
-## Choose batch or sessions mode
+## Choose the right execution mode
```

Comment:
This page feels a bit too long. Maybe we can move this part to a separate page?

Reply:
Let's take a look after it rebuilds with all the other changes.  A couple of sections were removed, so maybe that'll make it more reasonable.